src/Components/Projects/Projects.tsx

```diff
@@ -59,7 +59,6 @@ const Projects: React.FC = () => {
     axios
       .get(`${baseUrl}/Project/manager`, { headers: requestHeaders })
       .then((response) => {
-        console.log(response?.data[0].title);
         setProjects(response?.data);
```

Comment:
@serhj 
I noticed now that the projects array is inside the data object and it should be like this:
response?.data?.data

It was working with me when I first tested the UI.